packages/fluxible-router/lib/createNavLinkComponent.js

```diff
@@ -292,12 +289,24 @@ module.exports = function createNavLinkComponent (overwriteSpec) {
             this.dispatchNavAction(e);
         },
         render: function () {
-            var href = this._getHrefFromProps(this.props);
-            var activeClass = this.props.activeClass;
-            var activeStyle = this.props.activeStyle;
-            var activeElement = this.props.activeElement;
+            var props = this.props;
+            var href = this._getHrefFromProps(props);
+            if (!href) {
+                if (__DEV__) {
+                    throw new Error('NavLink created with invalid href \'' + props.href +
+                        '\'or unresolvable routeName \'' + props.routeName);
+                } else {
+                    var logError = (this.context.logger && this.context.logger.error) || console.error;
+                    logError('Error: Invalid NavLink, skip rendering...', props);
+                    return null;
```

Comment:
instead of returning `null`, I'm thinking if we should just render empty href, and when user click, should return  https://github.com/yahoo/fluxible/blob/master/packages/fluxible-router/lib/createNavLinkComponent.js#L262 and fallback to native anchor behavior with empty href.